### PR TITLE
New

### PR DESCRIPTION
**Pull request by sumantsingh70**
I just added my profile card to the website.
This was my first contribution hope i will learn more.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
...
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/sumant_singh__" target="_blank">@sumant_singh__</a>
+          </p>
+          <p class="about">Just starting my Journey with open-source</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
```
Comment (TenTraicion) [inline index.html:130]: kindly add resources!

**Review by TenTraicion — CHANGES_REQUESTED** [review]
kindly add resources in your card!

File: index.html [hunk 2]
```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:118]: kindly replace `TEMPLATE: MAKE A COPY` with your username

File: index.html [hunk 3]
```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Resource 3</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:141]: kindly replace `END TEMPLATE` with your username!

File: index.html [hunk 4]
```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
...
+          </p>
+          <p class="about">Just starting my Journey with open-source</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="_blank" title="First Resource">Resource 1</a>
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Second Resource">Resource 2</a>
```
Comment (TenTraicion) [inline index.html:133]: kindly add resources!

File: index.html [hunk 5]
```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#" target="_blank" title="Third resource">Resource 3</a>
```
Comment (TenTraicion) [inline index.html:136]: kindly add resources!

**Review by TenTraicion — CHANGES_REQUESTED** [review]
do the following changes and commit!

**Comment by sumantsingh70** [comment]
Now I completed all the resources.

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@sumantsingh70, Thanks for your Contribution. and Congrats on your first PR Merge in this project. Your Card will be live within ~2-3 minutes, which can be found: [Here](https://syknapse.github.io/Contribute-To-This-Project/)!

**Pull request merged**